src/analyzer.ts

```diff
@@ -198,15 +199,29 @@ export class BuildAnalyzer {
     //   1. The resolver stream, to resolve each file loaded via the analyzer
     //   2. The analyzer stream, to analyze app fragments for dependencies
     this._sourcesProcessingStream =
-        this._sourcesStream.pipe(new ResolveTransform(this))
+        this._sourcesStream
+            .on('error',
+                (err: Error) =>
+                    this._sourcesProcessingStream.emit('error', err))
+            .pipe(new ResolveTransform(this))
+            .on('error',
```

Comment:
I told you in person already, but that sucks :/ One day the world will be better...